src/Microsoft.OData.Client/DataServiceClientFormat.cs

```diff
@@ -4,14 +4,26 @@
 // </copyright>
 //---------------------------------------------------------------------
 
+
+
 namespace Microsoft.OData.Client
 {
+    #region namespaces
     using System;
     using System.Diagnostics;
     using System.Diagnostics.CodeAnalysis;
     using Microsoft.OData;
-    using Microsoft.OData.Edm;
-
+    using Edm;
+    using System.IO;
+    using System.Xml;
+    using System.Net;
+    using Microsoft.OData.Edm.Csdl;
+    using System.Collections.Generic;
+#if PORTABLELIB
+    using System.Threading.Tasks;
+#endif

```

Comment:
Portable Lib Support should be deprecated.

Reply:
Required in this case for supporting it in .net core